prompts/configs.go

```diff
@@ -66,19 +67,36 @@ func PromptForTargetConfig(targetName string, target *workflow.Target, existingC
 		}
 	}
 
-	var sdkClassName string
+	var sdkClassName string = ""
 	if !isQuickstart && output.Generation.SDKClassName != "" {
 		sdkClassName = output.Generation.SDKClassName
 	}
-	configFields := []huh.Field{
-		huh.NewInput().
-			Title("Name your SDK:").
-			Placeholder("your users will access SDK methods with <sdk_name>.doThing()").
-			Inline(true).
-			Prompt(" ").
-			Value(&sdkClassName),
+	_, err := charm.NewForm(
+		huh.NewForm(
+			huh.NewGroup(
+				huh.NewInput().
+					Title("Name your SDK:").
+					Description("your users will access SDK methods with myCompanySDK.doThing()").
+					// TODO: Are there languages where the default should be `my_company_sdk` python? What about terraform?
+					// TODO: If they are using the petstore spec we should default to `MyPetStoreSDK`
+					Placeholder("MyCompanySDK").
+					Prompt(" ").
+					// TODO: Can we do better in terms of validation? ie valid identifier
+					Validate(func(s string) error {
```

Comment:
@ryan-timothy-albert I would like your help with these TODOs as I'm sure it will take a lot less time to get a few pointers from you than muddle through it myself.

Reply:
There's not much to validate here. We pull all validation rules from openapi-generation. I don't think we should just make something up here, I always want openapi-generation to make the rules. The last thing we want is the CLI to diverge from the actual generator.

Reply:
Ideally we could pull in the language specific validator...